src/pudl/transform/eia.py

```diff
@@ -1149,133 +1113,131 @@ def fix_balancing_authority_codes_with_state(
     return plants
 
 
-@multi_asset(
-    ins={
-        asset_id: AssetIn()
-        for asset_id in [
-            "clean_boiler_fuel_eia923",
-            "clean_boiler_generator_assn_eia860",
-            "clean_boilers_eia860",
-            "clean_coalmine_eia923",
-            "clean_fuel_receipts_costs_eia923",
-            "clean_generation_eia923",
-            "clean_generation_fuel_eia923",
-            "clean_generation_fuel_nuclear_eia923",
-            "clean_generators_eia860",
-            "clean_ownership_eia860",
-            "clean_plants_eia860",
-            "clean_utilities_eia860",
-        ]
-        # This would be better, but currently results in a circular import error due to
-        # the use of occurrence_consistency in pudl.output.eia860. Need to untangle.
```

Comment:
this may no longer be relevant but we could move `occurrence_consistency` into helpers. it should probably live there anyway bc i've used it more like a helper and it is just in this module bc it was originally designed for the harvesting.

Reply:
That would be nice if it's easy. Does `occurence_consistency` have any weird dependencies I wonder? No it looks simple. I'll move it and try again.

Reply:
Well that didn't work unfortunately.  After moving `occurrence_consistency` then I got a new circular import on one of the BA code filling functions. And after moving that there was another one that was less clear, and I gave up.